server/store/middleware/infohash/config.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 The Chihaya Authors. All rights reserved.
+// Use of this source code is governed by the BSD 2-Clause license,
+// which can be found in the LICENSE file.
+
+package infohash
+
+import (
+	"github.com/chihaya/chihaya"
+	"gopkg.in/yaml.v2"
+)
+
+// Mode represents the mode of operation for an infohash scrape middleware.
+type Mode string
+
+const (
+	// ModeFilter makes the middleware filter disallowed infohashes from a
+	// scrape request.
+	ModeFilter = Mode("filter")
+
+	// ModeBlock maks the middleware block a scrape request if it contains
```

Comment:
I use IntelliJ - it shows spelling mistakes, but I ignore most of them because it doesn't know specialized terms like infohash.. I should start adding them to the dict and actually use the spelling check